aspnetcore/client-side/bower.md

```diff
@@ -14,231 +14,108 @@ uid: client-side/bower
+* Right-click the project in Solution Explorer and select **Manage Bower Packages** (alternatively from the main menu, **Project** > **Manage Bower Packages**).
 
-    ![manage bower packages](bower/_static/manage-bower-packages.png)
+* In the **Bower: \<project name\>** window, click the "Browse" tab, and then filter the packages list by entering `font-awesome` in the search box:
 
-* Confirm that the "Save changes to *bower.json*" checkbox is checked, select the desired version from the drop-down list, and click the Install button.
+ ![manage bower packages](bower/_static/manage-bower-packages.png)
 
-* Across the bottom status bar of the IDE, an *Installing "font-awesome" complete* message appears to indicate a successful installation.
+* Confirm that the "Save changes to *bower.json*" checkbox is checked. Select a version from the drop-down list and click the **Install** button. The **Output** window shows the installation details.
 
 ### Manual installation in bower.json
 
-* At the end of the `dependencies` section in *bower.json*, add a comma and type "font-awesome". Notice as you type that you get IntelliSense with a list of available packages. Select "font-awesome" from the list.
+Open the *bower.json* file and add "font-awesome" to the dependencies. IntelliSense shows the available packages. When a package is selected, the available versions are displayed. The images below are older and will not match what you see.
 
-    ![bower package explorer](bower/_static/add-package.png)
+![IntelliSense of bower package explorer](bower/_static/add-package.png)
 
-* Add a colon and then select the latest stable version of the package from the drop-down list. The double quotes will be added automatically.
+![bower version intellisense](bower/_static/version-intellisense.png)
 
-    ![bower version intellisense](bower/_static/version-intellisense.png)
-
-    > [!NOTE]
-    > Bower uses [semantic versioning](http://semver.org/) to organize dependencies. Semantic versioning, also known as SemVer, identifies packages with the numbering scheme \<major>.\<minor>.\<patch>. Intellisense simplifies semantic versioning by showing only a few common choices. The top item in the Intellisense list (4.6.3 in the example above) is considered the latest stable version of the package. The caret (^) symbol matches the most recent major version and the tilde (~) matches the most recent minor version.
-
-* Save the *bower.json* file.
+Bower uses [semantic versioning](http://semver.org/) to organize dependencies. Semantic versioning, also known as SemVer, identifies packages with the numbering scheme \<major>.\<minor>.\<patch>. Intellisense simplifies semantic versioning by showing only a few common choices. The top item in the Intellisense list (4.6.3 in the example above) is considered the latest stable version of the package. The caret (^) symbol matches the most recent major version and the tilde (~) matches the most recent minor version.
```

Comment:
I suggest changing the casing of "Intellisense" to "IntelliSense" all throughout, for consistency.